Makefile

```diff
@@ -72,7 +77,8 @@ test: manifests generate fmt vet envtest ## Run tests.
 
 .PHONY: e2e
 e2e:
-	go test -v -count=1 -tags e2e ./e2e
+	GOBIN=$(LOCALBIN) go install gotest.tools/gotestsum@${TAG}
```

Comment:
Could we move this to a dedicated makefile target? And then add that as a dependency of the `e2e`  target?

Reply:
Done